README.md

```diff
@@ -31,15 +33,15 @@ async def on_startup():
     print(f"This bot is owned by {bot.owner}")
 
 @message_command()
-    async def blurple_button(self, ctx):
-        await ctx.send("hello there", components=Button(ButtonStyles.BLURPLE, "A blurple button"))
+async def button(ctx):
```

Comment:
Surely this name would cause confusion between snek's `Button`, and the method `button`